base/database/utils.go

```diff
@@ -96,3 +97,8 @@ func PluckInt(tx *gorm.DB, columnName string) int {
 	}
 	return val
 }
+
+func IsValidUUID(uuid string) bool {
```

Comment:
Great, but please put this function to "base/utils/core.go", so we can call it as `utils.IsValidUUID(uuid string) bool`.